apps/web/src/pages/TalentEvents/TalentEvent/DetailsPage.tsx

```diff
@@ -20,10 +52,170 @@ interface TalentEventDetailsProps {
 }
 
 const TalentEventDetails = ({ query }: TalentEventDetailsProps) => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const intl = useIntl();
+
   const talentEvent = getFragment(TalentEventDetails_Fragment, query);
+  const developmentPrograms = unpackMaybes(talentEvent.developmentPrograms);
```

Comment:
Not able to get any values from this, is there a bug with `developmentPrograms`?

Reply:
Looks like our seeder justr doesn't seed any. You can get them working by manually adding them  in the DB.

![2025-04-01_11-50](https://github.com/user-attachments/assets/e563e7a3-a304-4a17-8b75-a2bf057833b0)

Reply:
Create issue for seeder https://github.com/GCTC-NTGC/gc-digital-talent/issues/13141.